src/unittests_parse_policies.c

```diff
@@ -1,59 +1,58 @@
 /*
-* Copyright 2018-2020 Redis Labs Ltd. and Contributors
-*
-* This file is available under the Redis Labs Source Available License Agreement
-*/
-#include <stdio.h>
-#include <stdlib.h>
-#include "parse_policies.h"
-#include "minunit.h"
+ * Copyright 2018-2020 Redis Labs Ltd. and Contributors
```

Comment:
@filipecosta90 its best to have a different PR for whitespaces changes.

Reply:
@danni-m I've simply applied clang-format. Checking why we're not doing it constantly...

Reply:
@danni-m we had this wrong formating due to missing clang-format clauses for test files. Fixed this on commit so we don't have this behaviour in the future https://github.com/RedisTimeSeries/RedisTimeSeries/pull/605/commits/29715b9eda6068e7325709976ef26f49e37c9a9a